pkg/okteto/preview.go

```diff
@@ -240,21 +245,40 @@ func (c *previewClient) ListEndpoints(ctx context.Context, previewName string) (
 		return nil, err
 	}
 
+	var endpointsMap = map[graphql.String]bool{}
+
+	for _, endpoint := range queryStruct.Response.Endpoints {
+		endpointsMap[endpoint.Url] = true
+	}
+
+	// @francisco - 2023//05/02
+	// The backend Endpoints field was being return empty in some older versions
+	// of the okteto api. Lets make sure we correctly resolve the endpoints from
+	// the known resources. All this below should be safe to remove in newer versions
+	// of the okteto cli
```

Comment:
Can we specify the version when we can remove this code? For example `when all okteto cluster are > 1.7`